package.json

```diff
@@ -17,7 +17,9 @@
     "stylelint": "stylelint '**/*.css'",
     "stylelint:fix": "pnpm run stylelint -- --fix",
     "test:integration": "turbo test:integration --log-order=stream --",
+    "test:integration:stream": "turbo test:integration --ui=stream --",
     "test:unit": "turbo test:unit --log-order=stream --",
+    "test:unit:stream": "turbo test:unit --ui=stream --",
```

Comment:
If you find it more useful to use stream here, just update the existing commands. Likely everyone would rather what you prefer.

Reply:
I prefer to keep both for a period of time until I choose one or the other.

Reply:
nah because we'll never get rid of it in the future. I'd rather we keep it clean. Just change the original commands. If someone complains then we'll know if we need more commands.

Reply:
Ok, merged options.